src/lib/pcb-soup-to-svg.ts

```diff
@@ -85,53 +118,127 @@ function pcbSoupToSvg(soup: AnySoupElement[]): string {
   }
 }
 
-function createPcbComponent(component: any, transform: any): string {
+function createSvgElement(item: AnySoupElement, transform: any): any {
+  switch (item.type) {
+    case 'pcb_component':
+      return createPcbComponent(item, transform);
+    case 'pcb_trace':
+      return createPcbTrace(item, transform);
+    case 'pcb_plated_hole':
+    case 'pcb_smtpad':
```

Comment:
i forgot to mention this but smtpads aren't holes so this is a bit weird

Reply:
It was getting handled here, https://github.com/tscircuit/circuit-to-svg/blob/feat/add-svgson/src/lib/pcb-soup-to-svg.ts#L175

But I will make a different function itself